cirq-google/cirq_google/engine/engine_client.py

```diff
@@ -378,25 +376,17 @@ async def delete_program_async(
 
     delete_program = duet.sync(delete_program_async)
 
-    @deprecated_parameter(
-        deadline='v1.4',
-        fix='Use `processor_id` instead of `processor_ids`.',
-        parameter_desc='processor_ids',
-        match=lambda args, kwargs: _match_deprecated_processor_ids(args, kwargs),
-        rewrite=lambda args, kwargs: rewrite_processor_ids_to_processor_id(args, kwargs),
-    )
     async def create_job_async(
         self,
         project_id: str,
         program_id: str,
         job_id: Optional[str],
-        processor_ids: Optional[Sequence[str]] = None,
+        processor_id: str = "",
```

Comment:
This shouldn't be optional now that there's a single field to specify the processor.